Code/Tools/FBuild/FBuildCore/Graph/ObjectNode.cpp

```diff
@@ -1454,6 +1454,39 @@ bool ObjectNode::BuildArgs( const Job * job, Args & fullArgs, Pass pass, bool us
                 }
             }
         }
+	
+	//remove static analyzer from clang preprocessor
+	if ( pass == PASS_PREPROCESSOR_ONLY )
+	{
+	   if ( isClang )
+	   {
+	      if ( StripTokenWithArg( "--analyze", token, i ) )
```

Comment:
StripTokenWithArg will strip the passed in string AND the next token.
I'm not familiar with all these args - is that correct?  Do all of them take a required next argument?